### PR TITLE
Portfolio project mobile version

### PR DESCRIPTION
**Pull request by Ogaga01**
Added index.html
Added linters
Added images
Added styling

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./style.css">
+        <title>Mobile Version</title>
+    </head>
+
+    <body>
+        <main class="mobile">
+            <header class="toolbar">
+                <p class="john">John Doe</p>
```
Comment (prolajumokeoni) [inline index.html]: - [x] Kindly wrap your logo in an a tag instead of a p tag, as this is part of the project requirements 👍🏿

File: index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                <nav>
+                    <img class="menu" src="./images/Icon-Menu.png" alt="menu icon">
+                </nav>
+            </header>
+            <section class="about">
+                <h1>Laura Jong</h1>
+                <h2>Software Developer</h2>
+                <p class="first-para">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+                <img class="divider" src="./images/Vector4.svg" alt="divider">
```
Comment (prolajumokeoni) [inline index.html:23]: - [x] Kindly add the languages, frameworks,  skills and icons after this section this template has them immediately  after the paragraph
LOGO AND ICONS

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Ogaga01, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design



## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by JuliCarracedo — CHANGES_REQUESTED** [review]
Hi @Ogaga01, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights

- [✔️](https://emojipedia.org/check-mark/) Great job applying the changes requested!
- [✔️](https://emojipedia.org/check-mark/) Professional PR and README
- [✔️](https://emojipedia.org/check-mark/) Your portfolio looks really good

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ogaga01** [comment]
@JuliCarracedo Hello Julian. I couldn't see the optional changes you requested on Github for my project review. Do you mind highlighting them again or sending them to me here on slack?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./style.css">
+        <title>Mobile Version</title>
+    </head>
+
+    <body>
+        <main class="mobile">
+            <header class="toolbar">
+                <h2 class="john">John Doe</h2>
```
Comment (JuliCarracedo) [inline index.html]: - [ ] Kindly wrap this in an `a` tag. Logos are meant to work as links to the homepage.
- [ ] Remember to use your own name/nickname here. It's your portfolio after all

File: style.css [hunk 4]
```diff
@@ -0,0 +1,138 @@
+body {
+  background: #191765;
+  background-image: url(./images/top-circles.svg);
+  background-repeat: no-repeat;
+  max-width: 375px;
```
Comment (JuliCarracedo) [inline style.css]: - [ ] Kindly remove this line of code to make your app fit any mobile screen size. This is causing a gap to appear on the right for any screen bigger than 375px
We usually discourage limiting your app's responsiveness by adding this kind of setting.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,138 @@
+body {
```
Comment (JuliCarracedo) [inline style.css:1]: - [ ] There's a gap on top and to the left that can be resolved by adding a margin of 0.

Comment (Ogaga01) [reply style.css:1]: @JuliCarracedo Can you explain this further?

File: index.html (hunk 3 above)
Comment (Ogaga01) [reply index.html]: @JuliCarracedo What should I wrap in an <a> tag?

Comment (JuliCarracedo) [reply index.html]: `<a><h2 class="john">John Doe</h2></a>`

File: style.css (hunk 5 above)
Comment (JuliCarracedo) [reply style.css:1]: When I deployed I could see that the body had a margin that was separating your content from the screen's limit by like 4 pixels on the top and left sides. I already closed my machine so I can't really show you what it looked like.

File: style.css (hunk 4 above)
Comment (Ogaga01) [reply style.css]: @JuliCarracedo Since it's the mobile version, isn't there supposed to be a max-width?

**Review by omar-muhamad — APPROVED** [review]
# Approved :heavy_check_mark:
### Hi @,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## (Highlights) Good Points: :+1:
- You documented your work professionally. :heavy_check_mark:
- Github flow is used correctly. :heavy_check_mark:
- All linters checks are ok. :heavy_check_mark:

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- [x] [Optional] Please fix all element widths because I can see that some elements overflowing like the image below. You can use a percentage instead of fixed values but make sure that matches the design. Using percentage on width makes your design more responsive.

| Figma design  | Your design|
| ------------- | ------------- |
| image  | image |

- [ ] Please Edit your README file by using this [template](https://github.com/microverseinc/readme-template) and fill the important section in it with your data.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**